docs/eventing/event-registry.md

```diff
@@ -14,7 +14,7 @@ type information in the cluster data store.
 
 ## Before you begin
 
-1. Read about the [broker](./broker/) and [trigger](./triggers/) objects.
+1. Read about the [broker](./broker/) and [trigger](./broker/triggers/) objects.
```

Comment:
this links are broken, do we care to fix them in this PR or maybe later we do a pass over all broken links?